recipes/pyecharts/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "pyecharts" %}
+{% set version = "2.0.4" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pyecharts-{{ version }}.tar.gz
+  sha256: 14503e688d4de8560e3d41c19db1b472d9a0208e25dd860626e1e8c665d12d20
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - jinja2
+    - prettytable
+    - python >=3.6
+    - simplejson
+
+test:
+  imports:
+    - pyecharts
+    - pyecharts.charts
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/pyecharts/pyecharts
+  summary: Python options, make charting easier
+  description: |
+    Apache ECharts is easy-to-use, highly interactive and highly performant
+    javascript visualization library under Apache license. Since its first public
+    release in 2013, it now dominates over 74% of Chinese web front-end market.
+    Yet Python is an expressive language and is loved by data science community.
+    Combining the strength of both technologies, pyecharts is born.
+  license: MIT
+  license_file: LICENSE
+  doc_url: https://pyecharts.org/
+  dev_url: https://github.com/pyecharts/pyecharts
+
+extra:
+  recipe-maintainers:
```

Comment:
We need approval from these maintainers to be listed.

Reply:
This project, at the moment, is mainly maintained by me (sunhailin-Leo).  
A few other maintainers are no longer working on maintenance, but I've kept them on the list of project maintainers out of respect for their contributions, so I hope you can understand.(chenjiandongx, chfw, kinegratii) https://github.com/pyecharts/pyecharts#-author

Reply:
We will only add them as maintainer here if they consent to it. Please remove them otherwise. Conda-forge maintenance and upstream maintenance is independent.

Reply:
Even if the past maintainers I listed aren't working on maintenance right now, or if they aren't available to make approvals right now, that shouldn't stop me, as the current maintainer, from listing them on the repository's maintainer list.

Reply:
This is not about upstream maintenance but about maintenance of this conda package. These are independent things. We don't want to spam anyone without their approval with PR review requests.